### ISSUE TITLE
How to assign a specific IP address to the container

### ISSUE THREAD

**Issue by naotoy** [socketplane/socketplane#150]
Can I assign a specific IP address to the container?

Current socketplane assigns unused IP address in 10.1.0.0/16 (if network is default) automatically.
In migration baremetal server to container, I want to assign same IP address as old baremetal server to new container.

**Comment by WIZARD-CXY** [comment]
I think you may need to hack the ipam request ip part a little

**Comment by mavenugo** [comment]
Yes. We dont have that functionality today. But the ipam module is easy to hack this in :-)

**Comment by naotoy** [comment]
Thank you for your help. I'll try it.
